clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/AbstractGoogleCachingAgent.groovy

```diff
@@ -22,11 +22,11 @@ import com.google.api.client.googleapis.batch.BatchRequest
 import com.google.api.client.http.HttpRequest
 import com.google.api.client.http.HttpRequestInitializer
 import com.google.api.services.compute.Compute
+import com.netflix.servo.util.VisibleForTesting
```

Comment:
I think this is supposed to be `com.google.common.annotations.VisibleForTesting`.